phac-nml/irida-next
# I18n-tasks configuration

## What does this PR do and why?
Installed & configured [i18n-tasks](https://github.com/glebm/i18n-tasks). 

## Screenshots or screen recordings
N/A

## How to set up and validate locally
We need to verify all the translations remain the same to the end user. 

1. Open a terminal.
2. Navigate to the `irida-next` project.
3. Run the command `i18n-tasks health`.
4. Verify that there are no missing or unused translations. 

Lastly, verify the tests pass as a [minitest](/irida-next-core/test/i18n_test.rb) has been added to the project.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
